cli/script/command-parser.ts

```diff
@@ -273,7 +273,11 @@ var argv = yargs.usage(USAGE_PREFIX + " <command>")
             .demand(/*count*/ 1, /*max*/ 2)  // Require one non-optional and one optional argument.
             .example("login", "Logs in to the CodePush server")
             .example("login --accessKey mykey", "Logs in on behalf of the user who owns and created the access key \"mykey\"")
+            .example("login --proxy proxyURL", "Logs in with the specified proxy url")
```

Comment:
To help illustrate the fact that this parameter expects a complete URL, can we replace "proxyURL" with an example like "https://someproxy.com:455"?